KarhooUISDK/Screens/BookingScreen/AddressBar/BookingAddressBarPresenter.swift

```diff
@@ -90,19 +90,28 @@ final class BookingAddressBarPresenter: AddressBarPresenter {
         }
     }
 
-    func prebookSelected() {
+    func prebookSelected(optionalDoneCallback: (() -> Void)?) {
```

Comment:
Isn't optional implied by the question mark? Could we rename this to just callback for consistency with other methods in the SDK @AleksanderWedrychowskiKarhoo?